src/views/teachers/landing/landing.jsx

```diff
@@ -85,7 +85,7 @@ var Landing = injectIntl(React.createClass({
                         <FlexRow className="guides-and-tutorials">
                             <div>
                                 <a href="/go">
-                                    <img src="/svgs/teachers/resources.svg" alt="resources icon" />
+                                    <img src="/svgs/teachers/cards.svg" alt="cards icon" />
```

Comment:
Sorry about that @St19Galla. Did not realize we had a consistent image ratio on this page. 

Here is a new SVG with the correct ratio: 
[v2-cards.svg.zip](https://github.com/LLK/scratch-www/files/1188513/v2-cards.svg.zip)